src/test/rgw/bucket_notification/api.py

```diff
@@ -229,6 +229,8 @@ def del_config(self, notification=None):
 
 
 test_path = os.path.normpath(os.path.dirname(os.path.realpath(__file__))) + '/../'
+build_path = test_path + '../../../build/'
```

Comment:
please remove these lines and add `def ceph()` function similar to the one in line 242, that invoke "call_rgw_ceph" from "test-rgw-call.sh"